src/main/java/net/fabricmc/loader/game/MinecraftGameProvider.java

```diff
@@ -51,7 +52,8 @@
 	public static final EntrypointTransformer TRANSFORMER = new EntrypointTransformer(it -> Arrays.asList(
 		new EntrypointPatchHook(it),
 		new EntrypointPatchBranding(it),
-		new EntrypointPatchFML125(it)
+		new EntrypointPatchFML125(it),
+		new EntrypointPatchBranding_1_8_9(it)
```

Comment:
Check for 1.8.9

Reply:
i did that in the class

Reply:
No?

Reply:
Line 80

Reply:
And if `class_669` doesn't exist you get a crash